mocha.el

```diff
@@ -71,11 +71,8 @@ From http://benhollis.net/blog/2015/12/20/nodejs-stack-traces-in-emacs-compilati
 
 (defun mocha-find-project-root ()
   "Find the root of the project."
-  (let ((root-files '("package.json" ".git" ".hg" ".svn")) (dir nil) (i 0))
-    (while (not dir)
-      (setq dir (locate-dominating-file default-directory (nth i root-files)))
-      (setq i (+ i 1)))
-    dir))
+  (let ((root-dir (f--traverse-upwards (f-exists? (f-expand "package.json" it)))))
```

Comment:
It's late and maybe I'm totally missing it, but in `(f-expand "package.json" it)` what is `it`